runners/spark/src/main/java/org/apache/beam/runners/spark/translation/EvaluationContext.java

```diff
@@ -149,10 +144,6 @@ public Dataset borrowDataset(PValue pvalue) {
     return dataset;
   }
 
-  <T> Iterable<? extends WindowedValue<?>> getPCollectionView(PCollectionView<T> view) {
```

Comment:
It remains the same as before the TranslationUtils.getSideInputs gets a new parameter "pviews" and is called inside the lambda, everything else is the same